### ISSUE TITLE
Max Intensity Value

### ISSUE THREAD

**Issue by luis-robotic** [ouster-lidar/ouster-ros#408]
Hello everyone.

I had a question concerning the Intensity field of the /ouster/points topic.
I am working with **ROS2 Humble** on an Ubuntu 22.04 machine and an Ouster **OS0** sensor.
I recorded some data and I realized that I don´t know which is the maximum value a point can return.
Is this value between 0-255?

Can you please help me with the issue?

**Comment by Samahu** [comment]
No, currently the intensity field in the ros-driver is mapped to the signal field (for historical reasons) which is defined as "Signal intensity photons in the signal return measurment", the field has no definitive maximum other than the field storage size of UINT16.

I recommend utilizing the reflectivity which is defined as the calibrated reflectivity and is scaled to the [0, 255].

At some point I want add an option to use that allows users to map reflectivity to the intensity instead of signal.

Hope this helps
